test/ProgressOnderwijsUtils.Tests/MaybeTests.cs

```diff
@@ -15,7 +14,8 @@ public void OkContainsTheValuePassedIn()
         {
             PAssert.That(() => Maybe.Ok("42").AsMaybeWithoutError<Unit>().Contains("42"));
             PAssert.That(() => Maybe.Ok("42").AsMaybeWithoutError<Unit>().Contains("hello world!") == false);
-            PAssert.That(() => Maybe.Ok("42").AsMaybeWithoutError<Unit>().Contains(default(string)) == false);
+            // ReSharper disable once RedundantCast
+            PAssert.That(() => Maybe.Ok((string?)"42").AsMaybeWithoutError<Unit>().Contains(default(string)) == false);

```

Comment:
```suggestion
            PAssert.That(() => Maybe.Ok("42").AsMaybeWithoutError<Unit>().Contains("37") == false);
```

of zoiets?

Reply:
Dan is hij niet echt anders dan de regel ervoor, toch?

Reply:
ah ja.
doh.